Develop->master

sychan opened the pull request:
Updates to get the docker image running and have a viable docker-compose config

sychan committed:
Running config for docker-compose, see comments in compose file about secret

sychan committed:
Fixed deprecated MAINTAINER directive.

kkellerlbl committed:
Merge pull request #1 from kbase/develop

Merge from parent

kkellerlbl commented:
Should wait till develop branch is on CI before merging, but should be safe to leave the PR open

sychan committed:
Change main Makefile so that kb-sdk generated files to match normal Server and Client names, remove references to old naming convention and have starman started directly and log to stdout

sychan committed:
Add flag to overwrite existing file in wget

sychan committed:
Identify the port that where we're listening

kkellerlbl committed:
Merge pull request #2 from kbase/develop

Develop

kkellerlbl committed:
Add NAMESPACE var

Add --define handle_namespace=$(NAMESPACE) to TPAGE_ARGS so that HandleServiceConstants.pm is generated properly.

kkellerlbl committed:
Merge pull request #37 from kkellerlbl/develop

Fix NAMESPACE generation

kkellerlbl committed:
Made handle-namespace an option in deployment config file.

kkellerlbl committed:
Merge pull request #38 from kkellerlbl/develop

TASK-1203 Made handle-namespace an option in deployment config file.

sychan committed:
Convert to using dockerize

sychan committed:
Merge pull request #40 from kbase/dockerize

Convert to using dockerize
